scripts/create-operator-bundle.sh

```diff
@@ -2,16 +2,16 @@
 
 # Initialization
 SCRIPT_DIR=$(dirname "$0")
-source $SCRIPT_DIR/init-env.sh
+source "$SCRIPT_DIR"/init-env.sh
 
 # Login to the registry
 docker login quay.io/testnetworkfunction
 
 # create the test operator
 rm -rf nginx-operator
 mkdir nginx-operator
-cd nginx-operator
-operator-sdk init --domain $REGISTRY_NAME --plugins helm
+cd nginx-operator || exit
```

Comment:
is this good ?
I think if the folder does not exist, there's no point in the next operation -operator-sdk-

Reply:
Yeah correct it makes the script `exit` if the folder doesn't exist.